index.js

```diff
@@ -1,47 +1,50 @@
 'use strict';
-const PCancelable = require('p-cancelable');
 const ManyKeysMap = require('many-keys-map');
+const domLoaded = require('dom-loaded');
+const pDefer = require('p-defer');
 
 const cache = new ManyKeysMap();
+const elementReady = (selector, options = {}) => {
+	const {
+		target = document,
+		stopOnDomReady = true
+	} = options;
```

Comment:
Thoughts on this pattern instead?


```js
const elementReady = (selector, {
	target = document,
	stopOnDomReady = true
} = {}) => {
	// Code
};
```

Reply:
Sure, that's fine too. It just has a tendency to become kinda messy when you have a lot of options.